README.md

```diff
@@ -1,4 +1,4 @@
 ## This is my Hello Microverse' project
 
 
-This is my first project at microverse
+This is my first project day 2

```

Comment:
- [ ] It's nice that you added a README file to your project, however for this project and all projects in microverse using the provided [README Template](https://github.com/microverseinc/readme-template) is __mandatory__ please use it in your project after customizing it by adding your project info _make sure to keep all required parts and sections of the README template_